buildspec.yml

```diff
@@ -0,0 +1,134 @@
+version: 0.2
+
+env:
+  variables:
+    FRAMEWORK_VERSION: '1.3.0'
+    CPU_FRAMEWORK_BINARY: 'https://files.pythonhosted.org/packages/71/64/49c5125befd5e0f0e17f115d55cb78080adacbead9d19f253afd0157656a/mxnet-1.3.0.post0-py2.py3-none-manylinux1_x86_64.whl'
+    CPU_PY_VERSION: '2'
+    CPU_INSTANCE_TYPE: 'ml.c4.xlarge'
+    GPU_FRAMEWORK_BINARY: 'https://files.pythonhosted.org/packages/52/8b/3f9cfe199e592b0723617e5d1919b83022f05cc359bd1885f7d1e9ce4758/mxnet_cu90mkl-1.3.0.post0-py2.py3-none-manylinux1_x86_64.whl'
+    GPU_PY_VERSION: '3'
+    GPU_INSTANCE_TYPE: 'ml.p2.xlarge'
+    ECR_REPO: 'sagemaker-test'
+    GITHUB_REPO: 'sagemaker-mxnet-container'
+    SETUP_FILE: 'setup_cmds.sh'
+    SETUP_CMDS: '#!/bin/bash\npip install --upgrade pip\npip install -U -e .\npip install -U -e .[test]'
+
+
+phases:
+  pre_build:
+    commands:
+      - start-dockerd
+      - |
+        ACCOUNT=$(aws sts get-caller-identity --query 'Account' --output text)
+        PREPROD_IMAGE="$ACCOUNT.dkr.ecr.$AWS_DEFAULT_REGION.amazonaws.com/$ECR_REPO"
+      # keep ssh connection alive when communicating with remote ec2 server during integ test
+      # largest connection idle time allowed: 30 seconds * 60 attempts = 30 minutes
+      - |
+        echo '  ServerAliveInterval 30' >> ~/.ssh/config
+        echo '  ServerAliveCountMax 60' >> ~/.ssh/config
+
+
+  build:
+    commands:
+      # install
+      - pip3 install -U -e .
+      - pip3 install -U -e .[test]
+
+      # run unit tests
+      - pytest test/unit
+
+      # Create mxnet-containe pip archive
```

Comment:
lol new typo

Reply:
maybe change to just 'Create pip archive' :)

Reply:
Ah!!!!!!!!!!!!!!

OK used 'Create pip archive'. Then I can just copy and paste to other buildspecs I think. Thanks!